requirements/requirements.txt

```diff
@@ -1,32 +1,118 @@
+aiofiles==23.1.0
 alabaster==0.7.13
+anyio==3.6.2
+argcomplete==2.1.2
+asgiref==3.6.0
+asttokens==2.2.1
 Babel==2.12.1
+backcall==0.2.0
 beautifulsoup4==4.12.2
+black==23.3.0
+bleach==6.0.0
+build==0.10.0
 certifi==2022.12.7
 charset-normalizer==3.1.0
+click==8.1.3
 colorama==0.4.6
+colorlog==6.7.0
 contourpy==1.0.7
+coverage==7.2.3
 cycler==0.11.0
+decorator==5.1.1
 distlib==0.3.6
 docutils==0.17.1
+executing==1.2.0
+fastapi==0.95.1
+fastjsonschema==2.16.3
 filelock==3.11.0
+flake8==6.0.0
+flake8-idom-hooks==0.6.4
+flake8-print==5.0.0
+Flake8-pyproject==1.2.3
+flake8-tidy-imports==4.8.0
+Flask==2.1.3
+Flask-Cors==3.0.10
+flask-sock==0.6.0
 fonttools==4.39.3
 furo==2022.4.7
+greenlet==2.0.1
+h11==0.14.0
+html5tagger==1.3.0
+httpcore==0.16.3
+httptools==0.5.0
+httpx==0.23.3
 idna==3.4
 imagesize==1.4.1
+importlib-metadata==6.4.1
+iniconfig==2.0.0
+ipython==8.12.0
+isort==5.12.0
+itsdangerous==2.1.2
+jaraco.classes==3.2.3
+jedi==0.18.2
 Jinja2==3.1.2
+jsonpatch==1.32
+jsonpointer==2.3
+keyring==23.13.1
 kiwisolver==1.4.4
 livereload==2.6.3
-MarkupSafe==2.1.2
+lxml==4.9.2
+markdown-it-py==2.2.0
+MarkupSafe==2.0.1
 matplotlib==3.7.1
+matplotlib-inline==0.1.6
+mccabe==0.7.0
+mdurl==0.1.2
+more-itertools==9.1.0
+multidict==6.0.4
+mypy==1.2.0
+mypy-extensions==1.0.0
+nox==2022.11.21
+noxopt==0.0.9
 numpy==1.24.2
 packaging==23.1
+parso==0.8.3
+pathspec==0.11.1
+pep8-naming==0.13.3
+pickleshare==0.7.5
 Pillow==9.5.0
+pkginfo==1.9.6
 platformdirs==3.2.0
+playwright==1.32.1
+pluggy==1.0.0
+prompt-toolkit==3.0.38
+pure-eval==0.2.2
+pycodestyle==2.10.0
+pydantic==1.10.7
+pydocstyle @ git+https://github.com/PyCQA/pydocstyle.git@bd4993345a241bd0d5128f21de56394b1cde3714
+pyee==9.0.4
+pyflakes==3.0.1
 Pygments==2.15.0
 pyparsing==3.0.9
+pyproject_hooks==1.0.0
+pytest==7.3.1
+pytest-asyncio==0.21.0
+pytest-mock==3.10.0
+pytest-rerunfailures==11.1.2
+pytest-timeout==2.1.0
```

Comment:
You've included test dependencies somehow. I recommend looking at the Dockerfile to see what's being installed. I believe it's just the `build-docs.txt` and the package itself.